templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>{{ Object|yesno:"Update, Create" }} new manufacturer</h1>

```

Comment:
The use of `Object|yesno:"Update, Create"` seems incorrect. It should be `object|yesno:"Update, Create"` with a lowercase 'o' for the variable name.